cookbooks/oceanic_extension/oceanic_extension.prm

```diff
@@ -0,0 +1,230 @@
+    set Maximum viscosity = 1e26
+
+    set Define thermal conductivities = true
+    set Thermal conductivities = 2.5
+    set Densities             =        3300,        3300,        2900,        3300
+    set Heat capacities       = 750.
+    set Thermal expansivities = 2e-5
+
+    # JBN - Add comment explaining what this value does and how it affects solution
+    set Viscosity averaging scheme = maximum composition
+
+    set Viscous flow law = composite
+
+    # JBN - add documenation about where these values come from
+    set Prefactors for dislocation creep          = 6.52e-16, 6.52e-16, 1.12e-10, 6.52e-16 
+    set Stress exponents for dislocation creep    =      3.5,      3.5,      3.4,      3.5
+    set Activation energies for dislocation creep =   530.e3,   530.e3,   497.e3,   530.e3
+    set Activation volumes for dislocation creep  =   18.e-6,   18.e-6,       0.,   18.e-6
+
+    set Prefactors for diffusion creep            = 2.37e-15, 1.00e-50, 1.00e-50, 2.37e-15
+    set Grain size exponents for diffusion creep  =      3.0,       0.,       0.,      3.0
+    set Activation energies for diffusion creep   =   375.e3,       0.,       0.,   375.e3
+    set Activation volumes for diffusion creep    =    2.e-6,       0.,       0.,    2.e-6
+
+    set Grain size = 5.e-3
+    set Angles of internal friction = 30.
+    set Cohesions = 20.e6, 20.e6, 20.e6, 20.e6
+
+    # The parameters below weaken the friction and cohesion by a
+    # a factor of 2 between plastic strain values of 0.5 and 1.5.
+    # To reduce the friction and cohesion by a factor of 4 simply
+    # set the strain weakening factors to 0.25.
+    set Strain weakening mechanism                  = plastic weakening with plastic strain only
+    set Start plasticity strain weakening intervals = 0.5
+    set End plasticity strain weakening intervals   = 1.5
+    set Cohesion strain weakening factors           = 0.5
+    set Friction strain weakening factors           = 0.5
+
+  end
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 9.81
+  end
+end
+
+
+# JBN - we can likely reduce some of the outputs below
+# Post processing
+subsection Postprocess
+set List of postprocessors = basic statistics, composition statistics, heat flux densities, heat flux statistics, mass flux statistics, material statistics, matrix statistics, pressure statistics, temperature statistics, topography, velocity statistics, visualization
+  subsection Visualization
+    set List of output variables      =  heat flux map, named additional outputs, strain rate, density, viscosity, melt fraction
+    subsection Material properties
+      set List of material properties = density, viscosity
+    end
+    set Time between graphical output =	100.e3
+    set Interpolate output            = true
+    set Write higher order output     = true
+    set Output format                 = vtu
+    set Number of grouped files       = 1
```

Comment:
Delete this line unless the number of grouped files had to be set to 1 for a specific reason?

Reply:
I used it to avoid having too many files when I run it on many cores. Does it not make sense?